.travis.yml

```diff
@@ -7,15 +7,13 @@ services:
   - mysql
 
 php:
-  - 7.1
   - 7.2
   - 7.3
 
 env:
   - SYMFONY_VERSION="3.4.*"
   - SYMFONY_VERSION="4.3.*"
-  - PHPUNIT_VERSION="6.*"
-  - PHPUNIT_VERSION="7.*"
```

Comment:
I would not drop support for version 7 yet as its still supported until Feb 2020:

https://phpunit.de/getting-started/phpunit-7.html